### PR TITLE
Add uSBInternal transport type

### PR DESCRIPTION
**Pull request by cpiper**
Adding an additional transport type. If the X509 certificate
extension contains the uSBInternal transport, the server can now
parse that ability and display it on the list of registered
authenticators.

### CODE REVIEW

File: u2f-ref-code/java/src/com/google/u2f/server/data/SecurityKeyData.java [hunk 1]
```diff
@@ -18,7 +18,8 @@
     BLUETOOTH_BREDR("bt"),
     BLUETOOTH_LOW_ENERGY("ble"),
     USB("usb"),
-    NFC("nfc");
+    NFC("nfc"),
+    USB_INTERNAL("usb-int");
```
Comment (leshi) [inline u2f-ref-code/java/src/com/google/u2f/server/data/SecurityKeyData.java]: "usb-internal"

See the FIDO 1.2 javascript spec

File: u2f-ref-code/java/tests/com/google/u2f/TestVectors.java [hunk 2]
```diff
@@ -291,6 +291,35 @@
...
+      + "CzAJBgNVBAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVQQKDBhJbnRl"
+      + "cm5ldCBXaWRnaXRzIFB0eSBMdGQwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAAQu"
+      + "CXRfbg9BKnqEs2frC43LSmHR+jNrvs/jC9CiyPr3RzSoL8A0ElifTMEH+TLTFn6W"
+      + "HrZkwwgDR+UFYmwdXRXPoxcwFTATBgsrBgEEAYLlHAIBAQQEAwID2DAKBggqhkjO"
+      + "PQQDAgNHADBEAiBYtS8gXcl3LhvvkVlzYJgpD/tYUHae/Rw3z8lxQSeeXwIgDE2R"
+      + "yWxFfRpgeg0WsLVHu7Ll4oZUkBEuS5RgezrcrRg=";
+  protected static final X509Certificate TRUSTED_CERTIFICATE_EXTENDED_TRANSPORTS =
+      parseCertificateBase64(TRUSTED_CERTIFICATE_EXTENDED_TRANSPORTS_BASE64);
+
+  // Has Bluetooth Radio, Bluetooth Low Energy and NFC
```
Comment (leshi) [inline u2f-ref-code/java/tests/com/google/u2f/TestVectors.java]: the comment doesn't make it clear how this is different from TRUSTED_CERTIFICATE_MULTIPLE_TRANSPORTS_BASE64

File: u2f-ref-code/java/src/com/google/u2f/server/data/SecurityKeyData.java (hunk 1 above)
Comment (cpiper) [inline u2f-ref-code/java/src/com/google/u2f/server/data/SecurityKeyData.java]: Done

File: u2f-ref-code/java/tests/com/google/u2f/TestVectors.java (hunk 2 above)
Comment (cpiper) [inline u2f-ref-code/java/tests/com/google/u2f/TestVectors.java]: Updated.

**Comment by cpiper** [comment]
Updated patch with review comments.

**Pull request merged**